rules/cloud/aws/aws_delete_identity.yml

```diff
@@ -0,0 +1,22 @@
+title: Delete SES Identity
+id: 20f754db-d025-4a8f-9d74-e0037e999a9a
+status: experimental
+description: Identifies a SES identity is deleted. This may be an indicator of an adversary removing the account that carried out suspicious or malicious activities
+references:
+    - https://unit42.paloaltonetworks.com/compromised-cloud-compute-credentials/
+author: Janantha Marasinghe
+date: 2022/12/13
+tags:
+    - attack.defense_evasion
+    - attack.t1070
+logsource:
+    product: aws
+    service: cloudtrail
+detection:
+    selection:
+        eventSource: ses.amazonaws.com
+        eventName: delete-identity
```

Comment:
I'm not very familiar with cloud so. How often does deleting an identity occur on an AWS instance? And aren't any indicators we can use in the event itself to increase the suspiciousness?

Reply:
@blueteam0ps what about this discussion? Can't proceed until I get an answer.

Reply:
@nasbench  I do not think this will be be a high volume detection. I am unable to validate this further though.

Reply:
Thanks for the reply :D and for the rules